diff --git a/debian/control b/debian/control
index 4e1a046..74b98d2 100644
--- a/debian/control
+++ b/debian/control
@@ -14,9 +14,7 @@ Build-Depends: debhelper-compat (= 13),
                lintian,
                perl,
                pve-cluster (>= 5.0-32),
-               pve-doc-generator (>= 5.3-3),
                pve-qemu-kvm | qemu-utils,
-               zfsutils-linux,
 Standards-Version: 4.6.2
 Homepage: https://www.proxmox.com
 
