diff --git a/debian/control b/debian/control
--- a/debian/control
+++ b/debian/control
@@ -10,7 +10,6 @@ Build-Depends: debhelper-compat (= 13),
                libtest-mockmodule-perl,
                perl,
                pve-cluster (>= 9.0.1),
-               pve-firewall (>= 5.1.0~),
                pve-doc-generator (>= 5.3-3),
                libpve-access-control (>= 9.0.1),
 Standards-Version: 4.6.1
