From 78569d25e0ec26380610bd8612e033917fe98838 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Kamil=20Trzci=C5=84ski?= <ayufan@ayufan.eu>
Date: Sat, 14 Jan 2023 11:10:52 +0100
Subject: [PATCH] Remove proxmox-websocket-tunnel

---
 debian/control | 1 -
 1 file changed, 1 deletion(-)

diff --git a/debian/control b/debian/control
index 6e73fbd..dcc4b7e 100644
--- a/debian/control
+++ b/debian/control
@@ -17,7 +17,6 @@ Depends: libpve-access-control,
          libpve-common-perl (>= 7.2-6),
          libpve-storage-perl (>= 7.0-14),
          pve-cluster,
-         proxmox-websocket-tunnel,
          ${misc:Depends},
          ${perl:Depends},
 Breaks: libpve-common-perl (<< 4.0-89),
-- 
2.34.1

