From fbac50831d3340e05c5e5cc32a942ce9bdd52cbe Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Kamil=20Trzci=C5=84ski?= <ayufan@ayufan.eu>
Date: Sun, 15 Jan 2023 14:09:13 +0000
Subject: [PATCH] Remove pve-edk2-firmware

---
 debian/control | 2 --
 1 file changed, 2 deletions(-)

diff --git a/debian/control b/debian/control
index 5f10cd7..a3f70aa 100644
--- a/debian/control
+++ b/debian/control
@@ -18,7 +18,6 @@ Build-Depends: debhelper (>= 12~),
                perl,
                pkg-config,
                pve-cluster,
-               pve-edk2-firmware,
                pve-firewall,
                pve-ha-manager <!nocheck>,
                pve-qemu-kvm (>= 7.1~),
@@ -45,7 +44,6 @@ Depends: dbus,
          perl (>= 5.10.0-19),
          proxmox-websocket-tunnel,
          pve-cluster,
-         pve-edk2-firmware (>= 3.20210831-1),
          pve-firewall,
          pve-ha-manager (>= 3.0-9),
          pve-qemu-kvm (>= 3.0.1-62),
-- 
2.25.1

