From 757feaddb81e9c7e815beb715406dce1be591926 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Kamil=20Trzci=C5=84ski?= <ayufan@ayufan.eu>
Date: Sat, 14 Jan 2023 19:13:06 +0100
Subject: [PATCH] Remove proxmox-websocket-tunnel

---
 debian/control | 1 -
 1 file changed, 1 deletion(-)

diff --git a/debian/control b/debian/control
index 5f10cd7..491f8ac 100644
--- a/debian/control
+++ b/debian/control
@@ -43,7 +43,6 @@ Depends: dbus,
          libuuid-perl,
          libxml-libxml-perl,
          perl (>= 5.10.0-19),
-         proxmox-websocket-tunnel,
          pve-cluster,
          pve-edk2-firmware (>= 3.20210831-1),
          pve-firewall,
-- 
2.34.1

