From ec3dc48be46869bb5bc0c89929b40280448790e3 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Kamil=20Trzci=C5=84ski?= <ayufan@ayufan.eu>
Date: Sat, 14 Jan 2023 20:11:39 +0100
Subject: [PATCH] Remove pve-headers-5.15

---
 debian/control | 1 -
 1 file changed, 1 deletion(-)

diff --git a/debian/control b/debian/control
index 1e0ec8a..43d1cf6 100644
--- a/debian/control
+++ b/debian/control
@@ -29,7 +29,6 @@ Description: Proxmox Virtual Environment
 
 Package: pve-headers
 Architecture: all
-Depends: pve-headers-5.15
 Description: Default Proxmox VE Kernel Headers
  This is a metapackage which will install the kernel headers for the
  current default kernel.
-- 
2.34.1

