From 2cf9466af2a78d6c8864aa43e2f5028eab228543 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Kamil=20Trzci=C5=84ski?= <ayufan@ayufan.eu>
Date: Sat, 14 Jan 2023 20:11:19 +0100
Subject: [PATCH] Remove pve-kernel-5.15

---
 debian/control | 1 -
 1 file changed, 1 deletion(-)

diff --git a/debian/control b/debian/control
index 8a42cae..1e0ec8a 100644
--- a/debian/control
+++ b/debian/control
@@ -15,7 +15,6 @@ Depends: apt,
          openssh-client,
          openssh-server,
          proxmox-archive-keyring,
-         pve-kernel-5.15,
          pve-kernel-helper,
          pve-manager,
          pve-qemu-kvm,
-- 
2.34.1

