From 73584e0463d50f5a23ad1c3d16538f9a1da5164a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Kamil=20Trzci=C5=84ski?= <ayufan@ayufan.eu>
Date: Sat, 14 Jan 2023 20:11:50 +0100
Subject: [PATCH] Remove proxmox-archive-keyring

---
 debian/control | 1 -
 1 file changed, 1 deletion(-)

diff --git a/debian/control b/debian/control
index 43d1cf6..8892205 100644
--- a/debian/control
+++ b/debian/control
@@ -14,7 +14,6 @@ Replaces: proxmox-ve-3.10.0, proxmox-virtual-environment, pve-kernel
 Depends: apt,
          openssh-client,
          openssh-server,
-         proxmox-archive-keyring,
          pve-kernel-helper,
          pve-manager,
          pve-qemu-kvm,
-- 
2.34.1

