From 236b30bac5ee6707a92885742a260dc49f318da8 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Kamil=20Trzci=C5=84ski?= <ayufan@ayufan.eu>
Date: Sat, 14 Jan 2023 20:12:17 +0100
Subject: [PATCH] Remove pve-kernel-helper

---
 debian/control | 1 -
 1 file changed, 1 deletion(-)

diff --git a/debian/control b/debian/control
index 8892205..c017c27 100644
--- a/debian/control
+++ b/debian/control
@@ -14,7 +14,6 @@ Replaces: proxmox-ve-3.10.0, proxmox-virtual-environment, pve-kernel
 Depends: apt,
          openssh-client,
          openssh-server,
-         pve-kernel-helper,
          pve-manager,
          pve-qemu-kvm,
          qemu-server,
-- 
2.34.1

