From 0f6784253cb7bb0b23c503b77fa9b1eae5b66be2 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Kamil=20Trzci=C5=84ski?= <ayufan@ayufan.eu>
Date: Sun, 15 Jan 2023 12:38:54 +0000
Subject: [PATCH] multi-arch

---
 debian/lxcfs.links | 3 ++-
 debian/rules       | 4 +++-
 2 files changed, 5 insertions(+), 2 deletions(-)

diff --git a/debian/lxcfs.links b/debian/lxcfs.links
index 36e0af9..61f44cb 100644
--- a/debian/lxcfs.links
+++ b/debian/lxcfs.links
@@ -1 +1,2 @@
-usr/lib/x86_64-linux-gnu/lxcfs/liblxcfs.so usr/lib/x86_64-linux-gnu/liblxcfs.so
+#! /usr/bin/dh-exec
+usr/lib/${DEB_HOST_MULTIARCH}/lxcfs/liblxcfs.so usr/lib/${DEB_HOST_MULTIARCH}/liblxcfs.so
diff --git a/debian/rules b/debian/rules
index 9b6dc58..1c75872 100755
--- a/debian/rules
+++ b/debian/rules
@@ -3,13 +3,15 @@
 # Uncomment this to turn on verbose mode.
 #export DH_VERBOSE=1
 
+DEB_HOST_MULTIARCH ?= $(shell dpkg-architecture -qDEB_HOST_MULTIARCH) 
+
 %:
 	dh $@  --with autoreconf,quilt
 
 override_dh_install:
 	find $(CURDIR)/debian/lxcfs/lib/ -type d -empty -delete
 	sed -i "/dependency_libs/ s/'.*'/''/" \
-	    "$(CURDIR)/debian/lxcfs/usr/lib/x86_64-linux-gnu/lxcfs/liblxcfs.la"
+	    "$(CURDIR)/debian/lxcfs/usr/lib/$(DEB_HOST_MULTIARCH)/lxcfs/liblxcfs.la"
 
 override_dh_auto_configure:
 	dh_auto_configure -- --with-init-script=systemd \
-- 
2.25.1

