diff --git a/debian/control b/debian/control
index e19da61..95ca217 100644
--- a/debian/control
+++ b/debian/control
@@ -16,7 +16,7 @@ Build-Depends: bash-completion,
                linux-libc-dev,
                meson (>= 0.61),
                pkg-config,
-               systemd-dev,
+               libsystemd-dev,
                quilt,
 Standards-Version: 4.5.1
 Homepage: https://linuxcontainers.org
