From 7a72049740072049227fb61c3999ca3de8e9629c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Kamil=20Trzci=C5=84ski?= <ayufan@ayufan.eu>
Date: Sat, 14 Jan 2023 19:58:16 +0100
Subject: [PATCH] Remove pve-xtermjs

---
 debian/control | 1 -
 1 file changed, 1 deletion(-)

diff --git a/debian/control b/debian/control
index efd29900..acb1499b 100644
--- a/debian/control
+++ b/debian/control
@@ -83,7 +83,6 @@ Depends: apt-transport-https | apt (>= 1.5~),
          pve-firewall,
          pve-ha-manager,
          pve-i18n (>= 1.0-3),
-         pve-xtermjs (>= 4.7.0-1),
          qemu-server (>= 7.2-8),
          rsync,
          spiceterm,
-- 
2.34.1

