From 71d1c1f8913c554cb0d362aba6e46ab23ad290e1 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Kamil=20Trzci=C5=84ski?= <ayufan@ayufan.eu>
Date: Sat, 14 Jan 2023 12:15:03 +0100
Subject: [PATCH] Remove pve-doc-generator

---
 debian/control | 1 -
 1 file changed, 1 deletion(-)

diff --git a/debian/control b/debian/control
index 4684c5b..42a865b 100644
--- a/debian/control
+++ b/debian/control
@@ -10,7 +10,6 @@ Build-Depends: debhelper (>= 12),
                libpve-cluster-perl,
                libpve-common-perl,
                pve-cluster (>= 6.0-4),
-               pve-doc-generator (>= 5.3-3),
 Standards-Version: 4.5.1
 
 Package: pve-firewall
-- 
2.34.1

