diff --git a/debian/rules b/debian/rules
index addb32e..1bbb756 100755
--- a/debian/rules
+++ b/debian/rules
@@ -13,7 +13,7 @@ override_dh_install:
 	dh_apparmor -pswtpm --profile-name=usr.bin.swtpm
 
 override_dh_auto_test:
-	SWTPM_TEST_SECCOMP_OPT="--seccomp action=none" make -j4 check VERBOSE=1
+	#SWTPM_TEST_SECCOMP_OPT="--seccomp action=none" make -j4 check VERBOSE=1
 
 override_dh_clean:
 	dh_clean --exclude=man/man8/swtpm-localca.8
