From ad2fe3557e9d89e97e9079117b0552d5ef5c6443 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Kamil=20Trzci=C5=84ski?= <ayufan@ayufan.eu>
Date: Sat, 14 Jan 2023 19:11:15 +0100
Subject: [PATCH] Remove pve-lxc-syscalld

---
 debian/control | 1 -
 1 file changed, 1 deletion(-)

diff --git a/debian/control b/debian/control
index 26eb75b..c175e92 100644
--- a/debian/control
+++ b/debian/control
@@ -27,7 +27,6 @@ Depends: binutils,
          lxc-pve,
          pve-cluster (>= 4.0-8),
          pve-ha-manager (>= 3.0-9),
-         pve-lxc-syscalld,
          xz-utils,
          ${misc:Depends},
          ${perl:Depends},
-- 
2.34.1

