diff --git a/debian/control b/debian/control
index 2e8c876..bc884e2 100644
--- a/debian/control
+++ b/debian/control
@@ -10,7 +10,6 @@ Build-Depends: check,
                libdigest-hmac-perl,
                libfuse-dev (>= 2.9.2),
                libglib2.0-dev (>= 2.68),
-               libpve-access-control,
                libpve-apiclient-perl,
                libpve-common-perl (>= 5.0-28),
                libqb-dev (>= 2.0.1),
