commit e45d837b1663b327650c123288fda87068362f52
Author: Docker Compile <docker@compile.dev>
Date:   Sun Jan 15 17:26:16 2023 +0000

    0001-Support-for-arm64

diff --git a/debian/pve-qemu-kvm.links b/debian/pve-qemu-kvm.links
old mode 100644
new mode 100755
index 7d9c8b1..7661fe0
--- a/debian/pve-qemu-kvm.links
+++ b/debian/pve-qemu-kvm.links
@@ -1,4 +1,5 @@
-usr/bin/qemu-system-x86_64 usr/bin/kvm
+#! /usr/bin/dh-exec
+usr/bin/qemu-system-${ARCH} usr/bin/kvm
 
 # qemu-system-i386 and qemu-system-x86_64 provides the same hardware emulation
 usr/bin/qemu-system-x86_64 usr/bin/qemu-system-i386
diff --git a/debian/rules b/debian/rules
index 6d5f882..b8f0046 100755
--- a/debian/rules
+++ b/debian/rules
@@ -16,7 +16,7 @@ include /usr/share/dpkg/pkg-info.mk
 DEB_HOST_GNU_TYPE   ?= $(shell dpkg-architecture -qDEB_HOST_GNU_TYPE)
 DEB_BUILD_GNU_TYPE  ?= $(shell dpkg-architecture -qDEB_BUILD_GNU_TYPE)
 
-ARCH ?= $(shell dpkg-architecture -qDEB_HOST_GNU_CPU)
+export ARCH ?= $(shell dpkg-architecture -qDEB_HOST_GNU_CPU)
 
 PACKAGE=pve-qemu-kvm
 destdir := $(CURDIR)/debian/$(PACKAGE)
@@ -46,7 +46,7 @@ ${BUILDDIR}/config.status: configure
 	--localstatedir=/var \
 	--prefix=/usr \
 	--sysconfdir=/etc \
-	--target-list=$(ARCH)-softmmu,aarch64-softmmu \
+	--target-list=x86_64-softmmu,aarch64-softmmu \
 	--with-suffix="kvm" \
 	--with-pkgversion="${DEB_SOURCE}_${DEB_VERSION_UPSTREAM_REVISION}" \
 	--audio-drv-list="alsa" \
