From a3b308356dcf18cea7a67686fe789f81a8c6e407 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Kamil=20Trzci=C5=84ski?= <ayufan@ayufan.eu>
Date: Sat, 14 Jan 2023 12:12:30 +0100
Subject: [PATCH] Remove pve-doc-generator

---
 debian/control | 1 -
 1 file changed, 1 deletion(-)

diff --git a/debian/control b/debian/control
index 67cfbb6..fe02594 100644
--- a/debian/control
+++ b/debian/control
@@ -11,7 +11,6 @@ Build-Depends: debhelper (>= 12~),
                libpve-rs-perl (>= 0.7.3),
                lintian,
                pve-cluster,
-               pve-doc-generator (>= 5.3-3),
 Standards-Version: 4.5.1
 
 Package: pve-ha-manager
-- 
2.34.1

