From 20a0c3dc6e881fab17f6ff30cf1fc5572bf84f89 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Kamil=20Trzci=C5=84ski?= <ayufan@ayufan.eu>
Date: Sun, 15 Jan 2023 11:40:09 +0100
Subject: [PATCH] Remove pve-cluster

---
 debian/control | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/debian/control b/debian/control
index c3c1256..50be1d5 100644
--- a/debian/control
+++ b/debian/control
@@ -5,7 +5,6 @@ Maintainer: Proxmox Support Team <support@proxmox.com>
 Build-Depends: debhelper (>= 12~),
                libauthen-pam-perl,
                libnet-ldap-perl,
-               libpve-cluster-perl,
                libpve-common-perl (>= 6.0-11),
                libpve-rs-perl,
                libtest-mockmodule-perl,
@@ -13,7 +12,6 @@ Build-Depends: debhelper (>= 12~),
                libuuid-perl,
                lintian,
                perl,
-               pve-cluster (>= 6.1-4),
 Standards-Version: 4.5.1
 Homepage: https://www.proxmox.com
 
@@ -27,13 +25,11 @@ Depends: libauthen-pam-perl,
          libmime-base32-perl,
          libnet-ldap-perl,
          libnet-ssleay-perl,
-         libpve-cluster-perl,
          libpve-common-perl (>= 6.0-18),
          libpve-rs-perl (>= 0.4.3),
          libpve-u2f-server-perl (>= 1.0-2),
          liburi-perl,
          libuuid-perl,
-         pve-cluster (>= 6.1-4),
          ${misc:Depends},
          ${perl:Depends}
 Breaks: pve-manager (<< 7.0-15)
-- 
2.34.1

