From 84700cc78d63fb3542245cee1ce5ccaa9a039f95 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Kamil=20Trzci=C5=84ski?= <ayufan@ayufan.eu>
Date: Sat, 14 Jan 2023 11:08:11 +0100
Subject: [PATCH] Remove pve-doc-generator

---
 debian/control | 1 -
 1 file changed, 1 deletion(-)

diff --git a/debian/control b/debian/control
index 71ef42b..c3c1256 100644
--- a/debian/control
+++ b/debian/control
@@ -14,7 +14,6 @@ Build-Depends: debhelper (>= 12~),
                lintian,
                perl,
                pve-cluster (>= 6.1-4),
-               pve-doc-generator (>= 5.3-3)
 Standards-Version: 4.5.1
 Homepage: https://www.proxmox.com
 
-- 
2.34.1

