diff --git a/docs/Makefile b/docs/Makefile
index c57cbbc2..e06eef23 100644
--- a/docs/Makefile
+++ b/docs/Makefile
@@ -96,10 +96,10 @@ SPHINXBUILD   = sphinx-build
 BUILDDIR      = output
 
 ifeq ($(BUILD_MODE), release)
-COMPILEDIR := ../target/$(DEB_HOST_RUST_TYPE)/release
+COMPILEDIR := ../target/release
 SPHINXOPTS    += -t release
 else
-COMPILEDIR := ../target/$(DEB_HOST_RUST_TYPE)/debug
+COMPILEDIR := ../target/debug
 SPHINXOPTS    += -t devbuild
 endif
 
