diff --git a/Makefile b/Makefile
index 6435262f..75c696cc 100644
--- a/Makefile
+++ b/Makefile
@@ -149,7 +149,7 @@ clean-deb:
 	rm -f *.deb *.dsc *.tar.* *.buildinfo *.build *.changes
 
 .PHONY: dinstall
-dinstall: $(SERVER_DEB) $(SERVER_DBG_DEB) $(CLIENT_DEB) $(CLIENT_DBG_DEB)
+dinstall: $(SERVER_DEB) $(SERVER_DBG_DEB) $(CLIENT_DEB) $(CLIENT_DBG_DEB) $(DOC_DEB) $(RESTORE_DEB)
 	dpkg -i $^
 
 # make sure we build binaries before docs
