diff --git a/Makefile b/Makefile
index dfbaacab..6435262f 100644
--- a/Makefile
+++ b/Makefile
@@ -38,9 +38,9 @@ SUBCRATES != cargo metadata --no-deps --format-version=1 \
 
 ifeq ($(BUILD_MODE), release)
 CARGO_BUILD_ARGS += --release
-COMPILEDIR := target/$(DEB_HOST_RUST_TYPE)/release
+COMPILEDIR := target/release
 else
-COMPILEDIR := target/$(DEB_HOST_RUST_TYPE)/debug
+COMPILEDIR := target/debug
 endif
 
 ifeq ($(valgrind), yes)
